src/components/BeaconsForm.tsx

```diff
@@ -49,7 +50,11 @@ export const BeaconsForm: FunctionComponent<BeaconsFormProps> = ({
               <FormGroup errorMessages={errorMessages}>
                 <FormFieldset>
                   <FormLegendPageHeading>{pageHeading}</FormLegendPageHeading>
-                  {pageText}
+                  {typeof pageText === "string" ? (
```

Comment:
👍  Only suggestion is maybe to put this logic before the rendering of this to make it a bit more readable?

Happy either way though.

Reply:
Great suggestion!  Done 👍